core/src/components/datetime/datetime.tsx

```diff
@@ -2037,12 +2043,13 @@ export class Datetime implements ComponentInterface {
           if (popoverRef) {
             this.isTimePopoverOpen = true;
 
-            popoverRef.present(
+            popoverRef.presentFromTrigger(
```

Comment:
Using `presentFromTrigger` allows us to configure if the direct descendent should be focused when the popover is presented. We want to focus the direct descendent so the hours picker column is announced immediately (this happens on iOS).